blueprints/networking/glb-hybrid-neg-internal/variables.tf

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "prefix" {
+  description = "Prefix used for resource names."
+  type        = string
+  validation {
+    condition     = var.prefix != ""
+    error_message = "Prefix cannot be empty."
+  }
+}
+
+variable "project_names" {
+  description = "The project names."
+  type = object({
+    landing  = string
+    spoke_01 = string
+  })
+  default = {
+    landing  = "landing"
+    spoke_01 = "spoke-01"
+  }
+}
+
+variable "projects_create" {
+  description = "Parameters for the creation of the new project."
+  type = object({
+    billing_account_id = string
+    parent             = string
+  })
+  default = null
+}
+
+variable "region_configs" {
+  description = "The primary and secondary region parameters."
+  type = object({
+    r1 = object({
+      region_name = string
+      zone        = string
```

Comment:
can we implicitly set zone via a local, and simplify the region variable? it would also be nice to use the same convention FAST networking stages now use: `regions.primary` and `regions.secondary`.

Reply:
done